Fudan Kit/Course/GraduateCourseAPI.swift

```diff
@@ -54,6 +54,7 @@ public enum GraduateCourseAPI {
         let decoder = JSONDecoder()
         let dateFormatter = DateFormatter()
         dateFormatter.dateFormat = "YYYY-MM-dd"
+        dateFormatter.timeZone = TimeZone(abbreviation: "UTC")
```

Comment:
Is it possible to set to UTC+8 directly?

Reply:
Setting this timezone to UTC+8 causes the date to be 8 hours early. But I suppose there should be a simpler way.